agent/util-scripts/pbench-make-result-tb

```diff
@@ -10,11 +10,11 @@ pbench_bin="$(realpath -e ${script_path}/..)"
 
 function usage() {
     printf "usage:\n"
-    printf "${script_name} --result-dir=<pbench results dir> --target-dir=<where to put tar ball> [--help] [--user=<user>] [--prefix=<path>] [--xz-single-threaded=<0|1>]\n"
+    printf "%s" "${script_name} --result-dir=<pbench results dir> --target-dir=<where to put tar ball> [--help] [--user=<user>] [--prefix=<path>] [--xz-single-threaded=<0|1>]\n"
```

Comment:
The idea behind `printf` in general, even though we have been sloppy with its use up until now, is that the format string contains all the text that is constant with `%` modifiers which describe where the variables land among the text.

So in this case, we should really use (combining both `printf `lines):

```
printf "usage:\n%s --result-dir=<pbench results dir> --target-dir=<where to put tar ball> [--help] [--user=<user>] [--prefix=<path>] [--xz-single-threaded=<0|1>]\n" "${script_name}"
```

Note the `%s` holds the place for the variable `script_name` to be expanded, where we quote the variable to make sure spaces in the variable contents are kept together.